### PR TITLE
Support for additional_properties in zhmc_lpar_list module

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

### CODE REVIEW

**Comment by andy-maier** [comment]
Rebased on master after merging PR #854

File: docs/source/modules/zhmc_lpar_list.rst [hunk 1]
```diff
@@ -96,10 +96,26 @@ cpc_name
...
+  | **elements**: str
+
+
 full_properties
   If True, all properties of each LPAR will be returned. Default: False.
 
-  Note: Setting this to True causes a loop of 'Get Logical Partition Properties' operations to be executed.
+  Mutually exclusive with \ :literal:`additional\_properties`\ .
+
+  Note: Setting this to True causes a loop of 'Get Logical Partition Properties' operations to be executed. It is preferrable from a performance perspective to use the \ :literal:`additional\_properties`\  parameter instead.
```
Comment (simon-spinner) [inline docs/source/modules/zhmc_lpar_list.rst:118]:
```suggestion
  Note: Setting this to True causes a loop of 'Get Logical Partition Properties' operations to be executed. It is preferable from a performance perspective to use the \ :literal:`additional\_properties`\  parameter instead.
```

File: plugins/modules/zhmc_lpar_list.py [hunk 2]
```diff
@@ -114,12 +114,28 @@
...
+    required: false
+    default: []
   full_properties:
     description:
       - "If True, all properties of each LPAR will be returned.
         Default: False."
+      - Mutually exclusive with C(additional_properties).
       - "Note: Setting this to True causes a loop of 'Get Logical Partition
-        Properties' operations to be executed."
+        Properties' operations to be executed. It is preferrable from a
```
Comment (simon-spinner) [inline plugins/modules/zhmc_lpar_list.py:136]:
```suggestion
        Properties' operations to be executed. It is preferable from a
```

**Review by simon-spinner — APPROVED** [review]

File: plugins/modules/zhmc_lpar_list.py (hunk 2 above)
Comment (andy-maier) [reply plugins/modules/zhmc_lpar_list.py:136]: Occurs in other modules, too. Will be fixed with a separate PR for all of them.

File: docs/source/modules/zhmc_lpar_list.rst (hunk 1 above)
Comment (andy-maier) [reply docs/source/modules/zhmc_lpar_list.rst:118]: Occurs in other modules, too. Will be fixed with a separate PR for all of them.

**Pull request merged**